src/test/scala/firrtlTests/InfoSpec.scala

```diff
@@ -158,4 +159,17 @@ class InfoSpec extends FirrtlFlatSpec {
     result should containLine ("x <= _GEN_2 @[GCD.scala 17:22 GCD.scala 19:19]")
     result should containLine ("y <= _GEN_3 @[GCD.scala 18:22 GCD.scala 19:30]")
   }
+
+  "source locators for append option" should "use multiinfo" in {
+    val input = """circuit Top :
+                  |  module Top :
+                  |    input clock : Clock
+                  |    input in: UInt<32>
+                  |    output out: UInt<32>
+                  |    out <= in @[Top.scala 15:14]
+                  |""".stripMargin
+    val circuit = firrtl.Parser.parse(input.split("\n").toIterator, AppendInfo("myfile.fir"))
+    val result = circuit.serialize
```

Comment:
Seems like a good place for `containTree` (probably will need to import `FirrtlMatchers` or whatever it's called).
```suggestion
    val circuit = firrtl.Parser.parse(input.split("\n").toIterator, AppendInfo("myfile.fir"))
    val expectedInfos = Seq(FileInfo(StringLit("Top.scala 15:14")), FileInfo(StringLit("myfile.fir 6:9")))
    circuit should containTree { case MultiInfo(`expectedInfos`) => true }
    val result = circuit.serialize
```